tests/baselines/reference/bluebirdStaticThis.errors.txt

```diff
@@ -1,13 +1,75 @@
 tests/cases/compiler/bluebirdStaticThis.ts(5,15): error TS2420: Class 'Promise<R>' incorrectly implements interface 'Thenable<R>'.
   Property 'then' is missing in type 'Promise<R>'.
+tests/cases/compiler/bluebirdStaticThis.ts(7,12): error TS2323: Cannot redeclare exported variable 'try'.

```

Comment:
Yeah, Ryan and I were discussing the same thing. Weird we only have one test-case impacted by it... but I'll revisit the fix.